modules/samples/default/decision-tree/README.MD

```diff
@@ -19,4 +21,4 @@ Outcome
 
 * Running the above script will result in a generation of logistic regression model for the provided data-set.
```

Comment:
Fixed in https://github.com/wso2/product-ml/commit/f25ef0e5d5191348e2ef784499d156480a1d3dc0